build-aux/snap/snapcraft.yaml

```diff
@@ -386,12 +390,14 @@ parts:
             ;;
           lib/snapd/snap-bootstrap)
             TAGS+=(nomanagers)
-            case "${VERSION}" in
-              1337.*)
-                TAGS+=(withtestkeys withbootassetstesting faultinject statelocktrace)
+            ;& # fallthrough
+          lib/snapd/snapd)
+            case "${CRAFT_ARCH_BUILD_FOR}" in
+              arm64)
+                TAGS+=(withoptee)
                 ;;
             esac
-            ;;
+            ;& # fallthrough
```

Comment:
I find this case quite confusing and difficult to read, instead of the fallthrough maybe it would make sense to move the case of version 1337 out of the `case "${cmd}" ` clause and make it have its own clause.